src/simulation/simulation.jl

```diff
@@ -434,20 +424,47 @@ function _apply_warm_start!(model::DecisionModel)
 end
 
 """ Required update problem function call"""
-function _update_problem!(model::DecisionModel, sim::Simulation)
+function _update_model!(model::DecisionModel, sim::Simulation)
     _update_parameters(model, sim)
     _update_initial_conditions!(model, sim)
     return
 end
 
+function _update_simulation_state!(sim::Simulation)
+    sim_store = get_simulation_store(sim)
```

Comment:
unused

Reply:
This function is still WIP

Reply:
Needs a note indicating that opening the store is temp only.